Week1/homework/app.js

```diff
@@ -3,9 +3,173 @@
+      imgPath: 'https://i.dr.com.tr/cache/600x600-0/originals/0000000058245-1.jpg'
+    },
+    {
+      id: 9,
+      title: 'dava',
+      language: 'turkish',
+      author: 'franz_kafka',
+      desc:
+        "Dava, (Der Prozeß), bir sabah uyandığında kendisini sebebini anlamadığı bir suç nedeniyle dava edilmiş bulan Josef K. adlı kahramanın absürt durumunun anlatıldığı bir Franz Kafka romanıdır. Gerçekdışı niteliğiyle Kafka'nın şaşırtıcı yapıtları arasında çok önemli bir yeri olan Dava; tamamlanmamış bölümleriyle birlikte yazarın ölümünden iki yıl sonra, yakın arkadaşı Max Brod'un katkılarıyla, 1925'te yayımlanmıştır. ",
+      imgPath: 'https://www.iskultur.com.tr/dosyalar/2015/03/dava_10-255x420.jpg'
+    },
+    {
+      id: 10,
+      title: 'donusum',
+      language: 'turkish',
+      author: 'franz_kafka',
+      desc:
+        "Dönüşüm, Değişim veya Metamorfoz (Almanca özgün adı: Die Verwandlung), Franz Kafka'nın uzun öyküsü. İlk olarak 1915 yılında yayımlanmıştır.Kafka'nın en popüler eseri sayılabilir. Öykü, Gregor Samsa'nın bir sabah kendini dev bir böceğe dönüşmüş bulmasıyla başlar ve hayatındaki değişiklikleri anlatarak devam eder. ",
+      imgPath: 'https://i.dr.com.tr/cache/600x600-0/originals/0000000064022-1.jpg'
+    }
+  ];
+
+  function toCapitalize(text) {
+    return text
+      .toLowerCase()
+      .split(' ')
+      .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+      .join(' ');
+  }
 
   // Replace with your own code
-  console.log(bookTitles);
+  function render(container) {
+    // for (const book of bookTitles) {
+    //   const li = document.createElement('li');
+    //   li.innerText = book;
+    //   container.appendChild(li);
+    // }
+
+    for (const title of books) {
+      console.log(title);
+      const div = document.createElement('div');
+      const h3 = document.createElement('h3');
+      h3.innerText = toCapitalize(title.title.split('_').join(' '));
+      div.appendChild(h3);
+
+      const img = document.createElement('img');
+      img.setAttribute('src', title.imgPath);
+      div.appendChild(img);
+
+      const p = document.createElement('p');
+      const author = toCapitalize(title.author.split('_').join(' '));
+      p.innerText = `${author} - ${toCapitalize(title.language)}`;
+      div.appendChild(p);
+
+      const descPara = document.createElement('p');
+      descPara.innerText = title.desc;
+      descPara.className = 'description';
+      div.appendChild(descPara);
+
+      div.className = 'book';
+      container.appendChild(div);
+    }
+  }
+
+  function main() {
+    const root = document.getElementById('root');
+    const div = document.createElement('div');
```

Comment:
Rather than using a `div` as the container and further child `div` elements for each book, it would be better if you used an `ul` as the container and and `li` as the main element for each book. This makes your app better ARIA-compliant, because the book list can now be recognised by screen readers (for use by visually impaired users) as a list.